### PR TITLE
Rule 011

### PR DESCRIPTION
**Pull request by Jude-lextego**

### CODE REVIEW

File: Rule-011.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,9585 @@
...
+                "exec": [
+                  "const networkMaps = [\r",
+                  "    {\r",
+                  "   \"key\":\"55\",\r",
+                  "   \"active\":true,\r",
+                  "   \"cfg\":\"2.1.1\",\r",
+                  "   \"messages\":[\r",
+                  "      {\r",
+                  "         \"id\":\"002@1.0.0\",\r",
+                  "         \"host\":\"https://frmfaas.sybrin.com\",\r",
```
Comment (uche-lextego) [inline Rule-011.postman_collection.json:263]: @Jude-lextego Please remove all frmfaas urls

File: Rule-011.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,9585 @@
...
+              "options": {
+                "raw": {
+                  "language": "json"
+                }
+              }
+            },
+            "url": {
+              "raw": "http://gateway.openfaas:8080/function/off-transaction-monitoring-service-rel-1-0-0/quoteReply",
+              "protocol": "http",
+              "host": ["frmfaas", "sybrin", "com"],
```
Comment (uche-lextego) [inline Rule-011.postman_collection.json:406]: I'm not sure why your host has a bunch of "host": ["frmfaas", "sybrin", "com"]. and the complete URL is gateway.openfaas please can you fix it?

**Review by uche-lextego — APPROVED** [review]
LGTM

File: Rule-011.postman_collection.json (hunk 2 above)
Comment (Jude-lextego) [reply Rule-011.postman_collection.json]: Fixed boss

**Pull request merged**